core/src/main/java/org/sonarsource/sonarlint/core/WsHelperImpl.java

```diff
@@ -31,11 +34,15 @@
 import org.sonarsource.sonarlint.core.container.connected.validate.DefaultValidationResult;
 import org.sonarsource.sonarlint.core.container.connected.validate.PluginVersionChecker;
 import org.sonarsource.sonarlint.core.container.connected.validate.ServerVersionAndStatusChecker;
+import org.sonarsource.sonarlint.core.telemetry.TelemetryPayload;
+import org.sonarsource.sonarlint.core.util.ws.HttpConnector;
+import org.sonarsource.sonarlint.core.util.ws.PostRequest;
 import org.sonarsource.sonarlint.core.util.ws.WsResponse;
 
 import static com.google.common.base.Preconditions.checkNotNull;
 
 public class WsHelperImpl implements WsHelper {
+  private static final String TELEMETRY_ENDPOINT = "";
```

Comment:
I was a bit confused by this empty value at first. It's because it's "to be determined", right?